ui/components/multichain-accounts/multichain-account-menu/multichain-account-menu.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { fireEvent, act, waitFor } from '@testing-library/react';
+import { fireEvent, act } from '@testing-library/react';
```

Comment:
Updating these test to assert that props are being passed correctly but moved all uncontrolled tests to account list